client/Combatant/GetOrRollMaximumHP.ts

```diff
@@ -2,18 +2,31 @@ import { StatBlock } from "../../common/StatBlock";
 import { Dice } from "../Rules/Dice";
 import { CurrentSettings } from "../Settings/Settings";
 
-export function GetOrRollMaximumHP(statBlock: StatBlock) {
+export type VariantMaximumHP = "MINION" | "BOSS" | null;
```

Comment:
This should be an `enum` in order to avoid using string literals all over the place.

Reply:
Also, rather than having a `null` option, just make it an optional param (so it can be `undefined`).

Reply:
Or have a Default option, but the `null` option isn't optimal